### DISCUSSION TITLE
Is bit component framework agnostic ?

### DISCUSSION THREAD

**Discussion by BrunoBeraud** [teambit/bit#7432] [FAQ] [answered]
Hi, I was wondering if a bit component could be installed regardless of the app framework?

I would like to create a UI component library that could be installed on an Angular, React or VueJS application.

As I am more of an Angular developer, I would like to export a bit component from an Angular component.

Does bit use _Angular Element_ to export an angular component?

I suppose that if bit is able to do this, then my bit component must not have any peer dependencies

Thank !

**Comment by itaymendel** [comment, marked as answer]
hi,

bit does not "care" about which framework you use. if you are looking to build cross-framework design system, you can look into building it with web components. bit support lit and stencil for this.
you can read more in the documentation.
